schema/Core/Navigation.entityType.php

```diff
@@ -11,6 +11,12 @@
     'add' => '3.0',
     'label_field' => 'label',
   ],
+  'getPaths' => fn() => [
+    'browse' => 'civicrm/admin/menu',
+    'add' => 'civicrm/admin/menu/item?reset=1&action=add',
+    'update' => 'civicrm/admin/menu/item?reset=1&action=update&id=[id]',
+    'delete' => 'civicrm/admin/menu/item?reset=1&action=delete&id=[id]',
```

Comment:
@colemanw just noticed the delete dialog isn't actually using this form and this link doesn't seem to work for me

Reply:
@ufundo yea the jstree form wasn't using it either, so maybe it never actually worked even though the code stub is there in the form.